fiesta/apps/events/views/event.py

```
@@ -0,0 +1,324 @@
+import json
```

Ruff

Missing required import: `from __future__ import annotations`

Insert required import: `from __future__ import annotations`

(I002)